integrations/db/mariadb/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright (c) 2020 Oracle and/or its affiliates.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.helidon.integrations.db</groupId>
+        <artifactId>helidon-integrations-db-project</artifactId>
+        <version>2.1.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>mariadb</artifactId>
```

Comment:
This artifactId is wrong, see https://github.com/oracle/helidon/blob/master/DEV-GUIDELINES.md#package-and-module-structure

Reply:
Current approach is in profiles to allow users test it locally. I know that we must find anther way to run it in pipelines.

Reply:
I can see that. I don't think this means we should keep doing the same. @tomas-langer thoughts ?

Reply:
Think it's always good to give developer an option to simply test his changes locally. I'd like to keep both local and pipeline setup.

Reply:
artifactID is fixed.

Reply:
A simple shell script or docker-compose file or even some k8s yaml files would be easier/better than having all the logic coordinated by maven.